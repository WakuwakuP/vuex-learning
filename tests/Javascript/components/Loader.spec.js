import { shallowMount, createLocalVue } from '@vue/test-utils';
import Vuex from 'vuex';

import Loader from '@/components/Loader';

const localVue = createLocalVue();
localVue.use(Vuex);

describe('Loader', () => {
  // マウントテスト
  it('is a Vue Instance', () => {
    const wrapper = shallowMount(Loader);
    expect(wrapper.isVueInstance()).toBeTruthy();
  });

  // スナップショットテスト
  it('renders correctly', () => {
    const wrapper = shallowMount(Loader);
    expect(wrapper.element).toMatchSnapshot();
  });
});
