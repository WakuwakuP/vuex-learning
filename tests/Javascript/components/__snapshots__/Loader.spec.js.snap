// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader renders correctly 1`] = `
<div
  class="loader"
>
  <p
    class="loading__text"
  >
    Loading...
  </p>
   
  <div
    class="loader__item loader__item--heart"
  >
    <div />
  </div>
</div>
`;
